PhpcsChanged/Cli.php

```diff
@@ -234,6 +236,11 @@ function runSvnWorkflowForFile(string $svnFile, array $options, ShellOperator $s
 	$phpcsStandard = $options['standard'] ?? null;
 	$phpcsStandardOption = $phpcsStandard ? ' --standard=' . escapeshellarg($phpcsStandard) : '';
 
+	$warningSeverity = $options['warning-severity'] ?? null;
+	$phpcsStandardOption .= isset($warningSeverity) ? ' --warning-severity=' . escapeshellarg($warningSeverity) : '';
+	$errorSeverity = $options['error-severity'] ?? null;
+	$phpcsStandardOption .= $errorSeverity ? ' --error-severity=' . escapeshellarg($errorSeverity) : '';
```

Comment:
Is there a reason that the ternary for `$warningSeverity` uses `isset()` and the ternary for `$errorSeverity` uses an implicit boolean? I suspect it was just an accident, but we might as well pick a style and use it for both; I think they should both work fine, but the implicit boolean would also return false for an empty string, which might be slightly better.

Reply:
Yeah, there is a reason, I meant to use the `isset` everywhere after noticing an issue with `'0' ?` being false and thus `--warning-severity` was not set, while we wanted the `'0'` to allow to be passed, since it has a special meaning in PHPCS (do not report any warnings).

Thanks for catching this! I have also added new tests for catching this behaviour.